### ISSUE TITLE
[graphql] Move from HTTP level health checks to graphQL level health checks

### ISSUE THREAD

**Issue by emilwihlander** [Dsek-LTH/member-page#328]
## Short description of the feature
Apollo server supports different health checks, and if we move to graphQL level the bug where the gateway needs to be restarted first time should be resolved.

https://www.apollographql.com/docs/apollo-server/monitoring/health-checks/

Do this issue depend on another issue? Write Depends on #Issue_number here

[ ] Relates to frontend
[x] Relates to the backend
[ ] Requires modification of GraphQL
[ ] Requires modification of Database
[ ] Requires creating a new service
